stl/src/tzdb.cpp

```diff
@@ -1,15 +1,297 @@
+            DWORD _Last_error{0};
+            _Load_address(
+                _Icu_module, _Icu_functions._Pfn_ucal_getDefaultTimeZone, "ucal_getDefaultTimeZone", _Last_error);
+            _Load_address(_Icu_module, _Icu_functions._Pfn_ucal_openTimeZoneIDEnumeration,
+                "ucal_openTimeZoneIDEnumeration", _Last_error);
+            _Load_address(_Icu_module, _Icu_functions._Pfn_uenum_close, "uenum_close", _Last_error);
+            _Load_address(_Icu_module, _Icu_functions._Pfn_uenum_count, "uenum_count", _Last_error);
+            _Load_address(_Icu_module, _Icu_functions._Pfn_uenum_unext, "uenum_unext", _Last_error);
+            if (_Last_error == ERROR_SUCCESS) {
+                _Level = _Icu_api_level::__has_icu_addresses;
+            } else {
+                // reset last-error in-case a later GetProcAddress resets it
+                SetLastError(_Last_error);
+            }
+        }
+
+        _Icu_functions._Api_level.store(_Level, _STD memory_order_release);
+        return _Level;
+    }
+
+    _NODISCARD _Icu_api_level _Acquire_icu_functions() noexcept {
+        auto _Level = _Icu_functions._Api_level.load(_STD memory_order_acquire);
+        if (_Level <= _Icu_api_level::__detecting) {
+            _Level = _Init_icu_functions(_Level);
+        }
+
+        return _Level;
+    }
+
+    _NODISCARD int32_t __icu_ucal_getDefaultTimeZone(UChar* result, int32_t resultCapacity, UErrorCode* ec) noexcept {
+        const auto _Fun = _Icu_functions._Pfn_ucal_getDefaultTimeZone.load(_STD memory_order_relaxed);
+        return _Fun(result, resultCapacity, ec);
+    }
+
+    _NODISCARD UEnumeration* __icu_ucal_openTimeZoneIDEnumeration(
+        USystemTimeZoneType zoneType, const char* region, const int32_t* rawOffset, UErrorCode* ec) {
+        const auto _Fun = _Icu_functions._Pfn_ucal_openTimeZoneIDEnumeration.load(_STD memory_order_relaxed);
+        return _Fun(zoneType, region, rawOffset, ec);
+    }
+
+    _NODISCARD void __icu_uenum_close(UEnumeration* en) {
+        const auto _Fun = _Icu_functions._Pfn_uenum_close.load(_STD memory_order_relaxed);
+        return _Fun(en);
+    }
+
+    _NODISCARD int32_t __icu_uenum_count(UEnumeration* en, UErrorCode* ec) {
+        const auto _Fun = _Icu_functions._Pfn_uenum_count.load(_STD memory_order_relaxed);
+        return _Fun(en, ec);
+    }
+
+    _NODISCARD const UChar* __icu_uenum_unext(UEnumeration* en, int32_t* resultLength, UErrorCode* status) {
+        const auto _Fun = _Icu_functions._Pfn_uenum_unext.load(_STD memory_order_relaxed);
+        return _Fun(en, resultLength, status);
+    }
+
+    struct _Tz_link {
+        const char* _Target;
+        const char* _Name;
+    };
+
+    // FIXME: Likely not the final implementation just here to open a design discussion on
+    //        how to handle time_zone_link. See test.cpp for further details on the issue.
+    static const _Tz_link _Known_links[] = {

```

Comment:
`static` here is unnecessary (`const` already grants internal linkage, and we're within an unnamed namespace).

Reply:
Changed to `constexpr`